### ISSUE TITLE
Architecture of NN is basically BigGAN?

### ISSUE THREAD

**Issue by aman-tiwari** [grey-eye/talking-heads#3]
From the paper, they allude to the architecture of the neural network being very similar to [BigGAN](https://github.com/ajbrock/BigGAN-PyTorch). Would it be worth taking the placement of the self attention layers, resblock and  normalization layers from it? (see pages 17-19: https://arxiv.org/pdf/1809.11096.pdf , the AdaIN layers could go where the batchnorm+linear is used)

**Comment by MrCaracara** [comment]
Aha... I don't know why I didn't notice that the ResUp layers had two inputs before. But yeah, I guess that it makes sense for the AdaIN layers to go there! Good point!!

**Comment by aman-tiwari** [comment]
Did it help with results?
